docs/src/components/footer/nci-big.jsx

```diff
@@ -10,7 +10,9 @@ export const NCIBigFooter = ({ accountId, categoryId }) => {
 	return (
 		<footer className="usa-footer usa-footer--nci-big">
 			<div className="grid-container usa-footer__return-to-top">
-				<Link to="#top">Return to top</Link>
+				<Link to="#top">
+					<span>BACK TO TOP</span>
```

Comment:
Can we use a text-transform here?  Screen readers don't like all caps and might do strange things